src/components/AnimalDetail/AnimalDetail.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+// styles 
+import './AnimalDetail.css'
+
+const AnimalDetail = ({ image, name, latin, decsription, home, biotop, food, size, zoos, listOfZoos }) => {
+
+    return (
+        <>
+            {listOfZoos && (
```

Comment:
máš více možností co dělat, když data o listOfZoos nejsou. Jedna, tu co máš ty, je že se nezobrazí detail zvířete. To je legit. Co ale když seznam zoo není až tak podstatný? Chtěli bychom zobrazit zvíře, ale jen s tou částí o zoos počkat až budou data,